metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/5.6.2/package/scripts/service_check.py

```diff
@@ -32,7 +32,7 @@ def service_check(self, env):
         doc = '{"name": "Ambari Smoke test"}'
         index = "ambari_smoke_test"
 
-        Logger.info("Running Elastic search service check", file=sys.stdout)
+        Logger.info("Running Elastic search service check")
```

Comment:
Hi @cestella , would it be a good idea to leave this unchanged since @mmiklavc has PR #904 which addresses this issue?

Reply:
It would be normally, but I couldn't test it in kerberos without it (status check fails when kerberizing because of this).  As soon as master is fixed and #904 is in (has my +1 already), I'll merge.